editor/components/post-excerpt/index.js

```diff
@@ -16,17 +16,18 @@ import './style.scss';
 import { getEditedPostExcerpt } from '../../selectors';
 import { editPost } from '../../actions';
 
-function PostExcerpt( { excerpt, onUpdateExcerpt } ) {
+function PostExcerpt( { excerpt, onUpdateExcerpt, instanceId } ) {
+	const id = 'editor-post-excerpt-' + instanceId;
```

Comment:
Minor: normally we try to avoid sum for string concatenation and we use const id = `editor-post-excerpt-${ instanceId }`; instead.